api/core/v1/core.proto

```diff
@@ -162,14 +162,28 @@ message ObjectSelector {
     }
 }
 
-// TargetRef identifies Gateway API objects to apply a direct policy to.
+// PolicyTargetReference identifies Gateway API objects to directly apply policy to.
+// This is a copy of the upstream K8s Gateway API `targetRef` API.
```

Comment:
this comment is out of date, should be PolicyTargetReference API?

Reply:
Yeah, I think the link is out of date, is it this one? https://github.com/kubernetes-sigs/gateway-api/blob/b4f0307cc9269e73187300e72979e7e111ab74ab/apis/v1alpha2/policy_types.go#L39

Reply:
the intent was the call out the generic targetRef concept not the specific message it is using. will update though

Reply:
@npolshakova that link is the same as the one in the comment?
https://github.com/kubernetes-sigs/gateway-api/blob/b4f0307cc9269e73187300e72979e7e111ab74ab/apis/v1alpha2/policy_types.go#L34-L56

Reply:
That link works- I must have miscopied it when trying to find it!